examples/rpl_udp/rpl.c

```diff
@@ -96,42 +96,46 @@ void rpl_udp_init(int argc, char **argv)
 
         printf("Channel set to %" PRIi32 "\n", chan);
 
+        /* global address */
+        ipv6_addr_t global_addr, global_prefix;
+        ipv6_addr_init(&global_prefix, 0xabcd, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0);
```

Comment:
The RFC mentions a randomly generated Global ID. I would also prefer to use Unique Local IPv6 Addresses here, but maybe in a seperate PR? And maybe integrated into somewhere else than here? Some module to retrieve those randomly generated unique local addresses?